bin/weewx/drivers/ws1.py

```diff
@@ -33,8 +34,11 @@ def confeditor_loader():
 METER_PER_FOOT = 0.3048
 MILE_PER_KM = 0.621371
 
-DEFAULT_PORT = '/dev/ttyS0'
-DEBUG_READ = 0
+DEFAULT_SER_PORT = '/dev/ttyS0'
+DEFAULT_TCP_ADDR = '192.168.36.25'
+DEFAULT_TCP_PORT = 3000
+PACKET_SIZE = 50
+DEBUG_READ = False
```

Comment:
DEBUG_READ should be an integer, not a boolean.  the convention has been to use an integer so that verbosity levels can be specified, e.g., 0 for nothing, 1 for some, 2 for more, etc.